casts/free-podcasts-screencasts-pt_BR.md

```diff
@@ -33,8 +33,10 @@
 
 * [Data Hackers](https://datahackers.com.br/podcast) (podcast)
 * [Data Science Academy](http://datascienceacademy.com.br/blog/podcast-dsa/) (podcast)
+* [Let's Data](https://www.youtube.com/channel/UCS58303Tmd0wIKTddmRskdQ) (podcast)
 * [Pizza de Dados](https://pizzadedados.com) (podcast)
 * [Programação Dinâmica - Machine Learning em Python](https://www.youtube.com/playlist?list=PL5TJqBvpXQv5CBxLkdqmou_86syFK7U3Q) (screencast)
+* [Teste de Turing](https://castbox.fm/channel/Teste-de-Turing-id2546538) (podcast)
```

Comment:
- Complete with author to satisfy licensing
- Better multi broadcast platforms like `anchor.fm`. They have various target sources linking to common ones.

```suggestion
* [Teste de Turing](https://anchor.fm/testedeturing) - Erick Fonseca (podcast)
```